x-pack/plugins/synthetics/server/synthetics_service/formatters/format_configs.test.ts

```diff
@@ -82,11 +84,15 @@ const testBrowserConfig: Partial<MonitorFields> = {
 };
 
 describe('formatMonitorConfig', () => {
+  const logger = loggerMock.create();
+
   describe('http fields', () => {
     it('sets https keys properly', () => {
-      const yamlConfig = formatMonitorConfig(
+      const yamlConfig = formatMonitorConfigFields(
```

Comment:
Please add tests that cover both combining local and global params, as well as overwriting global params with local params.